.github/workflows/ci.yml

```diff
@@ -21,7 +21,6 @@ jobs:
           - 3.6
           - 3.7
         django-version:
-          - django~=2.2
```

Comment:
Tests were failing with django 2.2
I removed it because we have already started dropping django 2.2 support